clusterman/aws/client.py

```diff
@@ -74,11 +75,21 @@ def _init_session():
     global _session
 
     if not _session:
-        _session = boto3.session.Session(
-            staticconf.read_string("accessKeyId", namespace=CREDENTIALS_NAMESPACE),
-            staticconf.read_string("secretAccessKey", namespace=CREDENTIALS_NAMESPACE),
-            region_name=staticconf.read_string("aws.region"),
-        )
+        # when running clusterman's CLI, we want to use folks' personal AWS creds
+        # so that we don't need to distribute service user creds (and since we can't
+        # use Pod Identity outside of k8s :p)
+        if os.getenv("AWS_PROFILE"):
```

Comment:
I think gating on AWS_PROFILE might bite us when trying to move the actual clusterman instances to pod identity since there's no actual AWS_PROFILE, and i think even `AWS_PROFILE=default` would fail since it's not *technically* a profile to use default session creds. (unless we update this check to accept `AWS_PROFILE=''` as truthy too?)

Could we maybe give ourselves an easy way to transition by also checking USE_POD_IDENTITY envvar exist or something hacky like that?

Reply:
gah, I had tunnel-visioned on getting the CLI working and totally forgot that we also needed pod identity to work here - lemme refactor this a bit

Reply:
this should work now!